docs/proposals/drafts/2022023-remote-write-observatorium-api.md

```diff
@@ -0,0 +1,59 @@
+---
+toc: true
+title: 2022-02 Additional Metrics Write Endpoints
+---
+
+# Additional Metrics Write Endpoints Proposal
+
+* **Owners:**:
+  * `@marcolan018`
+
+* **Related Tickets:**
+  * `<JIRA, GH Issues>`
+
+* **Other docs:**
+  * [Exporting Metrics To Additional Consumers in RedHat Advanced Cluster Management for Kubernetes](https://docs.google.com/document/d/1t7GodF4s_V5MtFtjTeiT0UFeQuu7cuSzz9-7FEENDsQ) (internal Red Hat link)
+
+## TLDR
+
+We propose to support additional metrics write endpoints. The metrics write requests can be forwarded to additional backend endpoints, which support Prometheus remote write protocol, besides Thanos receivers.
+
+## Why
+Currently the Observatorium API always forwards the metrics to Thanos. Given that users have multiple consumers of metric data, they have a requirement for the collected metrics to be made available to these additional consumers. We need a way to let Observatorium API to forward the metrics to more than one targets.
+There are some other options to fulfill this requirement. But all of them have problems/restrictions:
+1. The clients send metrics to the additional consumers directly. This option is not workable in some scenarios. e.g. there is no network connection between the clients and the additional consumers. Also, some clients have very limited resources and this option will lead to more consumption of cpu and bandwidth.
+2. The users of the additional consumers can pull the metrics from Thanos side directly, but that's not a realtime solution. Also it will lead to heavy workload on Thanos if users consistently query massive metrics from it. 
```

Comment:
This reminds me of one more alternative which is described in https://rhobs-handbook.netlify.app/services/rhobs/analytics.md/#existing-production-pipelines - by leveraging `thanos replicate` tool (although that is also not a real-time solution).